pkgs/top-level/python-packages.nix

```diff
@@ -317,6 +317,14 @@ in {
 
   pycryptodome = callPackage ../development/python-modules/pycryptodome { };
 
+  pycryptodome_3_3 = self.pycryptodome.overridePythonAttrs (oldAttrs: rec {
```

Comment:
There is a policy of having only a single version of each package in `pythonPackages`